src/lib/konnectors_typed.ts

```diff
@@ -0,0 +1,39 @@
+import { Q } from 'cozy-client'
+
+/**
+ * Fetches the list of running jobs for konnectors
+ *
+ * @example
+ * // runningKonnectors = [{ state: 'running', message: { konnector: 'my-konnector' } }]
+ * const { data: runningKonnectors } = useQuery(
+ *  fetchRunningKonnectors.definition,
+ * fetchRunningKonnectors.options
+ * )
+ */
+export const fetchRunningKonnectors = {
+  definition: Q('io.cozy.jobs')
```

Comment:
definition should be wrapped into a function: 

```
definition: () => Q(...)
```

It is not yet mandatory, but would be pretty soon.

Reply:
Updated